source/proxy-dll/component/platform.cpp

```diff
@@ -7,13 +7,14 @@
 #include <utils/cryptography.hpp>
 #include "WinReg.hpp"
 #include "definitions/t8_engine.hpp"
+#include "component/config.hpp"
 
 namespace platform
 {
 	uint64_t bnet_get_userid()
 	{
 		static uint64_t userid = 0;
-		if (!userid) userid = utils::cryptography::xxh32::compute(utils::identity::get_sys_username());
+		if (!userid) userid = utils::cryptography::xxh32::compute(bnet_get_username());
```

Comment:
in case of changing name from config, userstats directory will change and people will start complaining.